### PR TITLE
[rust][fix] Fix move module crash

### PR DESCRIPTION
**Pull request by bowenyang007**
## Summary
Basically we didn't realize that ABI could be empty when uploading a new module. But it is possible. 

## Testing
No longer crashes
image

### CODE REVIEW

File: rust/processor/src/models/default_models/move_modules.rs [hunk 1]
```diff
@@ -86,8 +90,12 @@ impl MoveModule {
...
     ) -> Option<MoveModuleByteCodeParsed> {
+        let abi = mmb.abi.as_ref();
+        if abi.is_none() {
+            return None;
+        }
         Some(Self::convert_move_module(
-            mmb.abi.as_ref().unwrap(),
+            abi.unwrap(),
             mmb.bytecode.clone(),
         ))
```
Comment (CapCap) [inline rust/processor/src/models/default_models/move_modules.rs:100]: nit; I think you can use a `map` or a match on `abi`

Comment (bowenyang007) [reply rust/processor/src/models/default_models/move_modules.rs]: true. done.

**Comment by larry-aptos** [comment]
lint is broken.

**Comment by bowenyang007** [comment]
@larry-aptos it looks like it's from a previous PR...I fixed some random xclippy issue but seems like that wasn't enough.

**Comment by larry-aptos** [comment]
working on it.

**Comment by bowenyang007** [comment]
It's OK I fixed it.

**Comment by larry-aptos** [comment]
neat.

**Review by larry-aptos — APPROVED** [review]

**Review by NancySirois — APPROVED** [review]

**Pull request merged**